AISKU/src/Init.ts

```diff
@@ -95,15 +95,6 @@ try {
     //         ApplicationInsights, Telemetry
     //     };
     // }
-
```

Comment:
This code is no longer needed as it's auto injected as part of the rollup build step for the browser bundles